db/schema.rb

```diff
@@ -20,24 +20,24 @@
     t.string   "name",              null: false
     t.boolean  "is_paid"
     t.boolean  "is_permanent_paid"
-    t.datetime "created_at"
-    t.datetime "updated_at"
+    t.datetime "created_at",        null: false
```

Comment:
No clue. Just did a quick search on "user" in the migrations and nothing comes up before 2013 when the users table was renamed to members.